test/interface/test_sorting.py

```diff
@@ -109,43 +105,67 @@ def test_invalid_direction(self, session):
 
 class TestSortApplied(object):
 
-    @pytest.mark.usefixtures('multiple_bars_inserted')
+    """Tests that results are sorted only according to the provided
+    filters.
+
+    Excludes nonexplicit sorting from tests: do NOT test how different
+    RDBMS sort rows with the same values being ordered since this is not
+    consistent across RDBMS.
+
+    Excludes `NULL` sorting from tests: sorting fields containing `NULL`
```

Comment:
"Excludes nonexplicit sorting from tests" and "Excludes `NULL` sorting from tests" are confusing to me. 

Do you mean just that this test does not verify these things because they differ across RDBMSs?

Reply:
Yes, that's it.

I've rephrased it to (try to) make it readable. Basically:
- Don't test how rows with the same values are sorted.
- Don't test whether `NULL` field values are placed first or last when sorting.

Because these things differ across RDBMSs.